lib/web_console/middleware.rb

```diff
@@ -113,5 +119,13 @@ def respond_with_unavailable_session(id)
 
         Rack::Response.new(body, status, headers).finish
       end
+
+      def respond_with_unacceptable_request
+        status  = 406
+        headers = { 'Content-Type' => 'application/json; charset = utf-8' }
+        body    = { error: "A supported version is expected in the Accept header." }.to_json
```

Comment:
Can you move this message into a constant as the ones at the beginning of the file? I think we can extract those messages to locale files eventually.